pkgs/tools/misc/sysz/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenvNoCC,  fetchFromGitHub, lib, bash, gawk, fzf }:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "sysz";
+  version = "1.2.2";
+
+  src = fetchFromGitHub {
+    owner = "joehillen";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256:0idi94jb075jki1jkm7shmicv77ylm2xjpjq6c868j9w6pvnash0";
+  };
+
+  buildInputs = [
+    bash
+    gawk
+    fzf
+  ];
+
```

Comment:
fzf and gawk should be added with `makeWrapper` or `wrapProgram`, adding them to buildInputs doesn't do anything
```suggestion
```